infra/docker/bionic/Dockerfile

```diff
@@ -43,8 +43,8 @@ RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive \
     apt-get -qqy install doxygen graphviz wget zip unzip clang-format-8 python3 python3-pip python3-venv hdf5-tools pylint curl
 RUN apt-get update && apt-get -qqy install python3.8 python3.8-venv
-RUN pip3 install --upgrade pip
-RUN pip3 install yapf==0.22.0 numpy
+RUN python3.8 -m pip install --upgrade pip
+RUN python3.8 -m pip install yapf==0.22.0 numpy
```

Comment:
need to use pyhon3.8 but there is no pip3.8

Reply:
I need to check `yapf` working on format checker.

Reply:
I confirmed `yapf` works.